libraries/AP_Generator/AP_Generator_config.h

```diff
@@ -25,3 +25,7 @@
 #ifndef AP_GENERATOR_RICHENPOWER_ENABLED
 #define AP_GENERATOR_RICHENPOWER_ENABLED AP_GENERATOR_BACKEND_DEFAULT_ENABLED && HAL_PROGRAM_SIZE_LIMIT_KB > 1024
 #endif
+
+#ifndef AP_GENERATOR_LOWEHEISER_ENABLED
+#define AP_GENERATOR_LOWEHEISER_ENABLED AP_GENERATOR_BACKEND_DEFAULT_ENABLED
```

Comment:
&& HAL_PROGRAM_SIZE_LIMIT_KB > 1024

Reply:
for custom build only use HAL_PROGRAM_SIZE_LIMIT_KB > 2048
that would be my preference

Reply:
Done.